packages/discord.js/typings/index.d.ts

```diff
@@ -6857,6 +6899,12 @@ export interface ShardingManagerOptions {
   execArgv?: readonly string[];
 }
 
+export interface SharedInteractionResponseOptions {
```

Comment:
Could/should `fetchReply` be moved here?

Reply:
The intention is to deprecate the option entirely in favour of this, so no